### DISCUSSION TITLE
Proot distro doesn't work

### DISCUSSION THREAD

**Discussion by amiria703** [termux/termux-app#2479] [Q&A] [answered]
~ $ proot-distro install debian
[] Installing Debian (bullseye)...             [] Using cached rootfs tarball...
[] Checking integrity, please wait...          [] Extracting rootfs, please wait...
proot info: vpid 1: terminated with signal 11
[*] Writing '/data/data/com.termux/files/usr/var/lib/proot-distro/installed-rootfs/debian/etc/profile'...
find: ‘usr/lib/’: No such file or directory

**Comment by sylirre** [comment, marked as answer]
PRoot Distro is a bash script wrapper for `proot`. PRoot Distro does not work on devices where `proot` is faulty.
